tests/IBTK/var_c_helmholtz_2d.cpp

```diff
@@ -0,0 +1,261 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (c) 2017 - 2019 by the IBAMR developers
+// All rights reserved.
+//
+// This file is part of IBAMR.
+//
+// IBAMR is free software and is distributed under the 3-clause BSD
+// license. The full text of the license can be found in the file
+// COPYRIGHT at the top level directory of IBAMR.
+//
+// ---------------------------------------------------------------------
+
+// Config files
+#include <IBTK_config.h>
+#include <SAMRAI_config.h>
```

Comment:
```suggestion
#include <ibtk/config.h>
```
The headers in the PR work fine with autotools but are deprecated. `ibtk/config.h` works with both autotools and the (in-progress) CMake build system.